packages/components/src/styles/color-filters/toFilter.ts

```diff
@@ -0,0 +1,2 @@
+export const toFilter = (values: [number, number, number, number, number, number]): string =>
```

Comment:
would be more readable and safer to use if the parameters are named, eg. invert: number?

Reply:
`values` is the parameter, the rest are types.